File: h2/jruby/read/h2_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	h2_read.rb
#					Jun/09/2011
#
# --------------------------------------------------------
include Java
import org.h2.Driver
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
# --------------------------------------------------------
puts "*** 開始 ***"
str_connect = "jdbc:h2:file:/var/tmp/h2/cities"
conn = DriverManager.getConnection(str_connect,"SA", "")

sql_read_proc(conn)

conn.close()
puts "*** 終了 ***"
# --------------------------------------------------------

File: h2/jython/read/h2_read.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	h2_read.py
#					Jan/25/2011
#
# ----------------------------------------------------------------
#
import sys
import	string
from java.lang import	System
#
import java
from java.sql import DriverManager
#
# ----------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")
#
database = "file:/var/tmp/h2/cities"
url="jdbc:h2:" + database
user = "SA"
password = ""

conn = DriverManager.getConnection (url,user, password)

display_proc (conn)
#
conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------
